integ-test/src/test/resources/expectedOutput/calcite/explain_filter_push.json

```diff
@@ -0,0 +1,4 @@
+{
+  "logical": "LogicalProject(age=[$8])\n  LogicalFilter(condition=[>($3, 10000)])\n    LogicalFilter(condition=[<($8, 40)])\n      LogicalFilter(condition=[>($8, 30)])\n        CalciteLogicalIndexScan(table=[[OpenSearch, opensearch-sql_test_index_account]])\n",
+  "physical": "EnumerableCalc(expr#0..10=[{inputs}], age=[$t8])\n  CalciteEnumerableIndexScan(table=[[OpenSearch, opensearch-sql_test_index_account]], PushDownContext=[[FILTER:>($8, 30), FILTER:AND(<($8, 40), >($3, 10000))]])\n"
```

Comment:
We should include DSL query in physical plan if possible, e.g. codeGen=true.
DSL can aid in debugging. The DSL provides a clear representation of the query structure, allowing users to execute it directly against their index to verify results. For instance, if a filter isn't functioning as expected, running the DSL query can help pinpoint discrepancies and facilitate troubleshooting.

Reply:
added.